src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -49,7 +49,7 @@
   private final ConsumeService _consumeService;
   private final String _name;
 
-  public SingleClusterMonitor(Map<String, Object> props, String name) throws Exception {
+  private SingleClusterMonitor(Map<String, Object> props, String name) throws Exception {
```

Comment:
Since this class implements App changing the  visibility of this method  may break other users code.

Reply:
Resolved.